vaccine_feed_ingest/runners/ny/am_i_eligible_covid19vaccine_gov/parse.py

```diff
@@ -14,8 +15,13 @@
 with input_file.open() as fin:
     data = json.load(fin)
 
+# TODO: if we want to pull in pytz, we can add in the eastern tz here
```

Comment:
If we need it - feel free to add it (`poetry add <foo>`)! I will defer to @rajbot or @bryanculbertson on time-parsing advice in Python.

Reply:
Oh, cool, just didn't want to pull in new dependencies willy-nilly.  Adding that in.